sync/p2p_messages.md

```diff
@@ -139,6 +150,36 @@ Message is RLP encoded, fields:
 A signed transaction is serialized as a tagged and versioned
 [signed transaction](../serializations.md#signed-transaction).
 
+## MSG_BLOCK_TXS
+*(Tag = 13)
+
+Message is RLP encoded, fields:
+  - `Hash :: byte_array - The block we fetch TXs from`
+  - `Txs :: [byte_array] - List of serialized signed TXs`
+
+A signed transaction is serialized as a tagged and versioned
+[signed transaction](../serializations.md#signed-transaction).
+
+## MSG_KEY_BLOCK
+*(Tag = 10)
+
+Message is RLP encoded, fields:
+  - `KeyBlock :: byte_array - Serialized key block`
+
+The key block is [serialized](../serializations.md#key-block-header).
+
+## MSG_MICRO_BLOCK
+*(Tag = 11)
+
+Message is RLP encoded, fields:
+  - `MicroBlock :: byte_array - Serialized micro block`
+  - `Light :: bool - flag if micro block is light or normal`
+
+A normal micro block is [serialized](../serializations.md#micro-block-header).
+A light micro block is serialized using
```

Comment:
For light micro block:
* https://github.com/aeternity/epoch/blob/c247b32db2116aa9db700da01aeaae1c2944641e/apps/aecore/src/aec_peer_connection.erl#L932-L935
* https://github.com/aeternity/epoch/blob/c247b32db2116aa9db700da01aeaae1c2944641e/apps/aecore/src/aec_peer_connection.erl#L1165-L1176
* https://github.com/aeternity/epoch/blob/c247b32db2116aa9db700da01aeaae1c2944641e/apps/aecore/src/aec_peer_connection.erl#L959-L960

So description seems ok (even if referring to code).